### PR TITLE
Update install matrix for PT 1.0

### PR DESCRIPTION
**Pull request by JoelMarcey**

### CODE REVIEW

File: _get_started/installation/windows.md [hunk 1]
```diff
@@ -88,10 +88,10 @@ conda install pytorch cuda80 -c pytorch
 pip3 install torchvision
 ```
 
-#### CUDA 9.2
+#### CUDA 10.0
 
 ```bash
-conda install pytorch cuda92 -c pytorch
+conda install pytorch cuda10.0 -c pytorch
```
Comment (orionr) [inline _get_started/installation/windows.md]: Should this be `cuda100` given the first usage at the beginning of the changes? @pjh5 and @soumith?

File: _get_started/installation/windows.md [hunk 2]
```diff
@@ -166,40 +166,40 @@ _CUDA 8.x_
...
 # Python 3.5
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp35-cp35m-win_amd64.whl
+pip3 install http://download.pytorch.org/whl/cu10.0/torch-1.0.0-cp35-cp35m-win_amd64.whl
 pip3 install torchvision
 ```
 
 ```
 # Python 3.6
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp36-cp36m-win_amd64.whl
+pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp36-cp36m-win_amd64.whl
```
Comment (orionr) [inline _get_started/installation/windows.md]: Should this now be `cu92 -> cu100`?

File: _get_started/installation/linux.md [hunk 3]
```diff
@@ -145,10 +145,10 @@ conda install pytorch torchvision -c pytorch
 conda install pytorch torchvision cuda80 -c pytorch
 ```
 
-#### CUDA 9.2
+#### CUDA 10.0
 
 ```bash
-conda install pytorch torchvision cuda92 -c pytorch
+conda install pytorch torchvision cuda100 -c pytorch
```
Comment (orionr) [inline _get_started/installation/linux.md]: Are we agreed this should be cuda100?

File: _get_started/installation/linux.md [hunk 4]
```diff
@@ -203,57 +203,57 @@ pip install torch torchvision
...
 
 # if the above command does not work, then you have python 2.7 UCS2, use this command
-pip install http://download.pytorch.org/whl/cu80/torch-0.4.1-cp27-cp27m-linux_x86_64.whl
+pip install http://download.pytorch.org/whl/cu80/torch-1.0.0-cp27-cp27m-linux_x86_64.whl
 ```
 
 ```bash
 # Python 3.5
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp35-cp35m-linux_x86_64.whl
+pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp35-cp35m-linux_x86_64.whl
```
Comment (orionr) [inline _get_started/installation/linux.md]: Should this now be `cu92 -> cu100`?

File: _get_started/installation/linux.md [hunk 5]
```diff
@@ -203,57 +203,57 @@ pip install torch torchvision
...
 pip3 install torchvision
 ```
 
 ```bash
 # Python 3.6
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp36-cp36m-linux_x86_64.whl
+pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp36-cp36m-linux_x86_64.whl
 pip3 install torchvision
```
Comment (orionr) [inline _get_started/installation/linux.md]: Should this now be `cu92 -> cu100`?

File: _get_started/installation/linux.md [hunk 6]
```diff
@@ -203,57 +203,57 @@ pip install torch torchvision
...
 ```
 
 ```bash
 # Python 3.7
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1.post2-cp37-cp37m-linux_x86_64.whl
+pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0.post2-cp37-cp37m-linux_x86_64.whl
```
Comment (orionr) [inline _get_started/installation/linux.md]: Should this now be `cu92 -> cu100`?

File: _get_started/installation/windows.md [hunk 7]
```diff
@@ -166,40 +166,40 @@ _CUDA 8.x_
...
 ```
 
-_CUDA 9.2_
+_CUDA 10.0_
 
 
 ```
 # Python 3.5
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp35-cp35m-win_amd64.whl
+pip3 install http://download.pytorch.org/whl/cu10.0/torch-1.0.0-cp35-cp35m-win_amd64.whl
```
Comment (orionr) [inline _get_started/installation/windows.md]: Should this now be `cu10.0 -> cu100`?

File: _get_started/installation/windows.md [hunk 8]
```diff
@@ -166,40 +166,40 @@ _CUDA 8.x_
...
 # Python 3.6
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp36-cp36m-win_amd64.whl
+pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp36-cp36m-win_amd64.whl
 pip3 install torchvision
 ```
 
 ```
 # Python 3.7
-pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp37-cp37m-win_amd64.whl
+pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp37-cp37m-win_amd64.whl
```
Comment (orionr) [inline _get_started/installation/windows.md]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 9]
```diff
@@ -418,89 +412,89 @@ function commandMessage(key) {
...
+      "pip3 install http://download.pytorch.org/whl/cu80/torch-1.0.0-cp35-cp35m-win_amd64.whl<br/>pip3 install torchvision",
 
     "stable,pip,windows,cuda9.0,python3.5":
-      "pip3 install http://download.pytorch.org/whl/cu90/torch-0.4.1-cp35-cp35m-win_amd64.whl<br/>pip3 install torchvision",
+      "pip3 install http://download.pytorch.org/whl/cu90/torch-1.0.0-cp35-cp35m-win_amd64.whl<br/>pip3 install torchvision",
 
-    "stable,pip,windows,cuda9.2,python3.5":
-      "pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp35-cp35m-win_amd64.whl<br/>pip3 install torchvision",
+    "stable,pip,windows,cuda10.0,python3.5":
+      "pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp35-cp35m-win_amd64.whl<br/>pip3 install torchvision",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 10]
```diff
@@ -418,89 +412,89 @@ function commandMessage(key) {
...
+      "pip3 install http://download.pytorch.org/whl/cu80/torch-1.0.0-cp36-cp36m-win_amd64.whl<br/>pip3 install torchvision",
 
     "stable,pip,windows,cuda9.0,python3.6":
-      "pip3 install http://download.pytorch.org/whl/cu90/torch-0.4.1-cp36-cp36m-win_amd64.whl<br/>pip3 install torchvision",
+      "pip3 install http://download.pytorch.org/whl/cu90/torch-1.0.0-cp36-cp36m-win_amd64.whl<br/>pip3 install torchvision",
 
-    "stable,pip,windows,cuda9.2,python3.6":
-      "pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp36-cp36m-win_amd64.whl<br/>pip3 install torchvision",
+    "stable,pip,windows,cuda10.0,python3.6":
+      "pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp36-cp36m-win_amd64.whl<br/>pip3 install torchvision",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 11]
```diff
@@ -418,89 +412,89 @@ function commandMessage(key) {
...
+      "pip3 install http://download.pytorch.org/whl/cu80/torch-1.0.0-cp37-cp37m-win_amd64.whl<br/>pip3 install torchvision",
 
     "stable,pip,windows,cuda9.0,python3.7":
-      "pip3 install http://download.pytorch.org/whl/cu90/torch-0.4.1-cp37-cp37m-win_amd64.whl<br/>pip3 install torchvision",
+      "pip3 install http://download.pytorch.org/whl/cu90/torch-1.0.0-cp37-cp37m-win_amd64.whl<br/>pip3 install torchvision",
 
-    "stable,pip,windows,cuda9.2,python3.7":
-      "pip3 install http://download.pytorch.org/whl/cu92/torch-0.4.1-cp37-cp37m-win_amd64.whl<br/>pip3 install torchvision",
+    "stable,pip,windows,cuda10.0,python3.7":
+      "pip3 install http://download.pytorch.org/whl/cu92/torch-1.0.0-cp37-cp37m-win_amd64.whl<br/>pip3 install torchvision",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 12]
```diff
@@ -418,89 +412,89 @@ function commandMessage(key) {
...
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-shared-with-deps-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-shared-with-deps-latest.zip</a>",
 
     "stable,libtorch,linux,cuda9.0,cplusplus":
-      "# Currently only available as a Preview.",
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu90/libtorch-shared-with-deps-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu90/libtorch-shared-with-deps-latest.zip</a>",
 
-    "stable,libtorch,linux,cuda9.2,cplusplus":
-      "# Currently only available as a Preview.",
+    "stable,libtorch,linux,cuda10.0,cplusplus":
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu92/libtorch-shared-with-deps-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu92/libtorch-shared-with-deps-latest.zip</a>",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 13]
```diff
@@ -418,89 +412,89 @@ function commandMessage(key) {
...
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-macos-latest.zip</a>",
 
     "stable,libtorch,macos,cuda9.0,cplusplus":
-      "# Currently only available as a Preview.",
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu90/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu90/libtorch-macos-latest.zip</a>",
 
-    "stable,libtorch,macos,cuda9.2,cplusplus":
-      "# Currently only available as a Preview.",
+    "stable,libtorch,macos,cuda10.0,cplusplus":
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu92/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-macos-latest.zip</a>",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 14]
```diff
@@ -694,7 +688,7 @@ function commandMessage(key) {
     "preview,pip,linux,cuda9.0,python2.7":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html",
 
-    "preview,pip,linux,cuda9.2,python2.7":
+    "preview,pip,linux,cuda10.0,python2.7":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 15]
```diff
@@ -706,7 +700,7 @@ function commandMessage(key) {
     "preview,pip,linux,cuda9.0,python3.5":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html",
 
-    "preview,pip,linux,cuda9.2,python3.5":
+    "preview,pip,linux,cuda10.0,python3.5":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 16]
```diff
@@ -718,7 +712,7 @@ function commandMessage(key) {
     "preview,pip,linux,cuda9.0,python3.6":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html",
 
-    "preview,pip,linux,cuda9.2,python3.6":
+    "preview,pip,linux,cuda10.0,python3.6":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 17]
```diff
@@ -730,56 +724,56 @@ function commandMessage(key) {
     "preview,pip,linux,cuda9.0,python3.7":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html",
 
-    "preview,pip,linux,cuda9.2,python3.7":
+    "preview,pip,linux,cuda10.0,python3.7":
       "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 18]
```diff
@@ -730,56 +724,56 @@ function commandMessage(key) {
...
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu80/torch_nightly.html",
 
     "preview,pip,windows,cuda9.0,python3.5":
-      "# Preview Build Not Yet Available on Windows.",
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html",
 
-    "preview,pip,windows,cuda9.2,python3.5":
-      "# Preview Build Not Yet Available on Windows.",
+    "preview,pip,windows,cuda10.0,python3.5":
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 19]
```diff
@@ -730,56 +724,56 @@ function commandMessage(key) {
...
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu80/torch_nightly.html",
 
     "preview,pip,windows,cuda9.0,python3.6":
-      "# Preview Build Not Yet Available on Windows.",
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html",
 
-    "preview,pip,windows,cuda9.2,python3.6":
-      "# Preview Build Not Yet Available on Windows.",
+    "preview,pip,windows,cuda10.0,python3.6":
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 20]
```diff
@@ -730,56 +724,56 @@ function commandMessage(key) {
...
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu80/torch_nightly.html",
 
     "preview,pip,windows,cuda9.0,python3.7":
-      "# Preview Build Not Yet Available on Windows.",
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu90/torch_nightly.html",
 
-    "preview,pip,windows,cuda9.2,python3.7":
-      "# Preview Build Not Yet Available on Windows.",
+    "preview,pip,windows,cuda10.0,python3.7":
+      "pip install numpy torchvision_nightly<br />pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cu92/torch_nightly.html",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 21]
```diff
@@ -790,20 +784,20 @@ function commandMessage(key) {
     "preview,libtorch,linux,cuda9.0,cplusplus":
       "Download here: <br/><a href='https://download.pytorch.org/libtorch/nightly/cu90/libtorch-shared-with-deps-latest.zip'>https://download.pytorch.org/libtorch/nightly/cu90/libtorch-shared-with-deps-latest.zip</a>",
 
-    "preview,libtorch,linux,cuda9.2,cplusplus":
+    "preview,libtorch,linux,cuda10.0,cplusplus":
       "Download here: <br/><a href='https://download.pytorch.org/libtorch/nightly/cu92/libtorch-shared-with-deps-latest.zip'>https://download.pytorch.org/libtorch/nightly/cu92/libtorch-shared-with-deps-latest.zip</a>",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 22]
```diff
@@ -790,20 +784,20 @@ function commandMessage(key) {
...
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/nightly/cu80/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/nightly/cpu/libtorch-macos-latest.zip</a>",
 
     "preview,libtorch,macos,cuda9.0,cplusplus":
-      "# Currently only available CPU-only / no CUDA.",
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/nightly/cu90/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/nightly/cpu/libtorch-macos-latest.zip</a>",
 
-    "preview,libtorch,macos,cuda9.2,cplusplus":
-      "# Currently only available CPU-only / no CUDA.",
+    "preview,libtorch,macos,cuda10.0,cplusplus":
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/nightly/cu92/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/nightly/cpu/libtorch-macos-latest.zip</a>",
```
Comment (orionr) [inline assets/quick-start-module.js]: Should this now be `cu92 -> cu100`?

File: assets/quick-start-module.js [hunk 23]
```diff
@@ -418,89 +412,89 @@ function commandMessage(key) {
...
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-macos-latest.zip</a>",
 
     "stable,libtorch,macos,cuda9.0,cplusplus":
-      "# Currently only available as a Preview.",
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu90/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu90/libtorch-macos-latest.zip</a>",
 
-    "stable,libtorch,macos,cuda9.2,cplusplus":
-      "# Currently only available as a Preview.",
+    "stable,libtorch,macos,cuda10.0,cplusplus":
+      "Download here: <br/><a href='https://download.pytorch.org/libtorch/1.0.0/cu100/libtorch-macos-latest.zip'>https://download.pytorch.org/libtorch/1.0.0/cu80/libtorch-macos-latest.zip</a>",
```
Comment (orionr) [inline assets/quick-start-module.js]: Missed the cu80 in the link name.

File: _get_started/installation/linux.md (hunk 3 above)
Comment (JoelMarcey) [reply _get_started/installation/linux.md]: That's what I heard from @soumith

**Pull request merged**